src/actions/sso/index.ts

```diff
@@ -27,3 +31,14 @@ export { consumeResolutionRequest } from './resolution'
 export const cancelSSO: ThunkAction = dispatch => {
   return dispatch(navigationActions.navigateBackHome())
 }
+
+export const scheduleSuccessNotification: ThunkAction = dispatch => {
+  return dispatch(
+    scheduleNotification(
+      createInfoNotification({
+        title: I18n.t(strings.GREAT_SUCCESS),
+        message: 'Interaction successfully done finished',
```

Comment:
i18n please?

Reply:
also "done finished" :D

Reply:
this is a placeholder. Liz will come back with the actual copy to use when it is done finished.

Reply:
Fixed.